src/parse/kdbx4.rs

```diff
@@ -387,13 +387,13 @@ pub(crate) fn parse(data: &[u8], key_elements: &[Vec<u8>]) -> Result<Database, D
         .inner_random_stream
         .get_cipher(&inner_header.inner_random_stream_key)?;
 
-    let (root, meta) = xml_parse::parse_xml_block(&xml, &mut *inner_decryptor)?;
+    let kpxml = crate::xml_db::parse::parse(&xml, &mut *inner_decryptor)?;
```

Comment:
@sseemayer same comment as for `kxml`: I think we could use a more explicit name here.

Reply:
agreed and changed to `database_content`.